daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestCLIParsing.scala

```diff
@@ -151,7 +151,11 @@ class TestCLIParsing {
     }(ExitCode.BadExternalVariable)
   }
 
-  @Test def test_CLI_Parsing_SimpleParse_DFDL1197_fix(): Unit = {
+  // FIXME FIXME FIXME
```

Comment:
@stevedlawrence  I can't figure out why this test doesn't pass any more. 
The string it is searching for is there in the very verbose output. 
Can you take a quick look at this?

Reply:
Looks like the error now outputs
```
<Sequence><separator/><RepMinMax name='s1'>
```
Note the lowercase `s` in "separator". I guess changing the toString of the enums modified one of the parser names?

Reply:
Did not spot that. I guess the searches I was doing default to case insensitive. 

Thanks!